packages/astro/e2e/view-transitions.test.js

```diff
@@ -696,7 +697,7 @@ test.describe('View Transitions', () => {
 		await page.goto(astro.resolveUrl('/client-only-three'));
 		let msg = page.locator('#name');
 		await expect(msg).toHaveText('client-only-three');
-		await page.waitForTimeout(400); // await hydration
+		await page.waitForTimeout(1000); // await hydration
```

Comment:
Not a big fan of `waitForTimeout` 😅 In this case for hydration, could it be swapped to the `waitForHydrate` util?

We could also create utils that wait for events like `astro:page-load` and `astro:after-swap` so it runs more reliably.

Reply:
Thank you, good point, I wasn't aware of those!

Reply:
All `waitForTimeout`s gone.